bitnami/apisix/templates/dashboard/pdb.yaml

```diff
@@ -3,30 +3,30 @@ Copyright VMware, Inc.
 SPDX-License-Identifier: APACHE-2.0
 */}}
 
-{{- if .Values.dataPlane.pdb.create }}
+{{- if .Values.dashboard.pdb.create }}
 apiVersion: {{ include "common.capabilities.policy.apiVersion" . }}
 kind: PodDisruptionBudget
 metadata:
-  name: {{ template "apisix.data-plane.fullname" . }}
+  name: {{ template "apisix.dashboard.fullname" . }}
   namespace: {{ include "common.names.namespace" . | quote }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
     app.kubernetes.io/part-of: apisix
-    app.kubernetes.io/component: data-plane
+    app.kubernetes.io/component: dashboard
     {{- if .Values.commonLabels }}
     {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
     {{- end }}
   {{- if .Values.commonAnnotations }}
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 spec:
-  {{- if .Values.dataPlane.pdb.minAvailable }}
-  minAvailable: {{ .Values.dataPlane.pdb.minAvailable }}
+  {{- if .Values.dashboard.pdb.minAvailable }}
+  minAvailable: {{ .Values.dashboard.pdb.minAvailable }}
   {{- end }}
-  {{- if .Values.dataPlane.pdb.maxUnavailable }}
-  maxUnavailable: {{ .Values.dataPlane.pdb.maxUnavailable }}
+  {{- if .Values.dashboard.pdb.maxUnavailable }}
+  maxUnavailable: {{ .Values.dashboard.pdb.maxUnavailable }}
   {{- end }}
   selector:
     matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
```

Comment:
Neither here. You should also add the `podLabels`.

Reply:
Good catch! Done at https://github.com/bitnami/charts/pull/18456/commits/1fed0db0cec55398dcb476839ef72ba090a30091